CHANGELOG.md

```diff
@@ -6,6 +6,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] - 2024-03-21
+
+### Added
```

Comment:
Please just add a single line here, mentioning that you have added this example to Modulus. This level of detail is not needed and adding a single item is sufficient.